src/group.js

```diff
@@ -9,11 +9,11 @@ SVG.G = SVG.invent({
   extend: {
     // Move over x-axis
     x: function (x) {
-      return x == null ? this.transform('x') : this.transform({ x: x - this.x() }, true)
+      return x == null ? this.transform().e : this.translate(x - this.gbox().x, 0)
```

Comment:
Did you find a better solution for this? You mentioned that we could hget rid of gbox somehow

Reply:
Yep, testing now!